### ISSUE TITLE
Checking implementation

### ISSUE THREAD

**Issue by TrySound** [jwilsson/domtokenlist#5]
How about checking implementation for fully shim?

```js
var el = document.createElement('div');
el.classList.add('c1', 'c2');
if(el.className !== 'c1 c2') {
  // Bad implementation
}
```

**Comment by jwilsson** [comment]
Yeah, that's a good idea!

I'm just a bit unsure if all browsers will allow us to override their native implementation. Do you wanna send a PR and we'll test it out?

**Comment by TrySound** [comment]
Give me please access to repository for direct PR.

**Comment by TrySound** [comment]
[Here](http://stackoverflow.com/questions/11115998/is-there-a-way-to-add-remove-several-classes-in-one-single-instruction-with-clas/23316665#23316665) nice idea.

We will check classList if native DOMTokeList exists and modify it, not redefining.

**Comment by TrySound** [comment]
Also I want to work with minifying. This shim is "most wanted" for me. So it should be as small as possible.

**Comment by jwilsson** [comment]
Send some PRs and we'll continue the discussion there

**Comment by TrySound** [comment]
I just would like to develop this as a collaborator without forking.

**Comment by jwilsson** [comment]
Alright, adding you!

**Comment by TrySound** [comment]
Cool! Thanks! Will contribute :)

**Comment by jwilsson** [comment]
For future reference: Closed in 1bc6d2e.
